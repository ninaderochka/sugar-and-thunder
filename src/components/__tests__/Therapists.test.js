import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Therapists from '../Therapists';

test('Therapists component renders correctly', () => {
  const component = renderer.create(<Therapists />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Therapists component renders correctly and button click works', () => {
  const { getByText } = render(<Therapists />);
  const button = getByText('Book an Appointment');

  expect(getByText('Professional, licensed, and vetted therapists that you can trust')).toBeDefined();
  expect(getByText('Tap into the worldâ€™s largest network of licensed, accredited, and experienced therapists who can help you with a range of issues including depression, anxiety, relationships, trauma, grief, and more. with our therapists, you get the same professionalism and quality you would expect from an in-office therapist, but with the ability to communicate whenever and however you want.')).toBeDefined();
  expect(button).toBeDefined();
});
