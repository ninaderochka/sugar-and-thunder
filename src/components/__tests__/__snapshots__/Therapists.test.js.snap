// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Therapists component renders correctly 1`] = `
<div
  className="bg-light-yellow h-full w-screen max-h-full pb-10"
>
  <div
    className="w-5/6 flex flex-col space-y-12 m-auto"
  >
    <h1
      className="font-poppins font-normal text-5xl pt-20 uppercase w-9/12"
    >
      Professional, licensed, and vetted therapists that you can trust
    </h1>
    <img
      alt="therapists"
      className="w-full lg:w-2/4"
      src="./therapists.png"
    />
    <p
      className="font-poppins text-black/80 w-10/12"
    >
      Tap into the worldâ€™s largest network of licensed, accredited, and experienced therapists who can help you with a range of issues including depression, anxiety, relationships, trauma, grief, and more. with our therapists, you get the same professionalism and quality you would expect from an in-office therapist, but with the ability to communicate whenever and however you want.
    </p>
    <div
      className="top-2"
    >
      <button
        className="undefined font-poppins font-normal text-black text-2xl uppercase w-fit px-6 py-2.5 bg-button-blue leading-tight rounded shadow-md hover:bg-button-blue/90 hover:shadow-lg focus:bg-button-blue/90 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-button-blue/90 active:shadow-lg transition duration-150 ease-in-out"
        onClick={[Function]}
        type="button"
      >
        Book an Appointment
      </button>
    </div>
  </div>
</div>
`;
